### ISSUE TITLE
(NPC) Salvaged Demolisher / (NPC) Salvaged Siege Engine - Passenger Seats

### ISSUE THREAD

**Issue by durrrr** [DoctorCookie/WoWIL-Issues#51]
Description: This NPCs can be found in Ulduar.

Current behaviour: Sometimes when climbing on as a passenger you get stuck and cant exit or move in the seat, which requires a total restart of the game for you to get out of there. You also cant start the boss in that placement because you also cant attack.

Expected behaviour:
Passenger seat should not get you stuck

Steps to reproduce the problem:
1. Get in passenger seat of "Salvaged Demolisher"
2. Get in passenger seat of "Salvaged Siege Engine"
3. Try to get out, use spells or move

The Zone Wowhead: http://www.wowhead.com/ulduar
The 1st NPC Wowhead: http://www.wowhead.com/npc=33109/salvaged-demolisher
The 2nd NPC Wowhead: http://www.wowhead.com/npc=33060/salvaged-siege-engine

**Comment by DoctorCookie** [comment]
Any idea how it happens?

**Comment by durrrr** [comment]
No, and i tested more then a few times, it happens to random people at random time, so i couldn't figure it out
